src/parsers/eventParser.js

```diff
@@ -84,14 +113,51 @@ export const parseApiGwEvent = event => {
   return parsed_event;
 };
 
+export const parseSnsEvent = event => {
+  const new_sns_event = {};
+  new_sns_event['Records'] = [];
+  // Add order keys
+  for (const rec of event['Records']) {
```

Comment:
Create some util func to this code duplicate

Reply:
Ordering keys by list

Reply:
It has some specific logic in each one of the code, making a generic function will make to code unclear